python.d/neo.chart.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+# Description:  netdata python.d module for monitoring Neo4j
+# Author: Jerome Baton, copied from the example of Pawel Krupa (paulfantom)
+
+
+# if you want to add a chart:
+#     insert it in ORDER
+#     copy paste and rename a chart in CHARTS
+#     code your query to return two results per line (label , value)
+
+
+# Warning
+#    charts take time to appear in the browser, also dont forget to clean the cache
+#    see /var/log/netdata/error.log to get some info and know if your code is running
+#    (python.d exited with -1  is bad news)
```

Comment:
The comments here are probably unneeded, as the case is the same for any of the python.d plugins.